manual_run/data_provider.py

```diff
@@ -0,0 +1,73 @@
+import csv
+from enum import Enum
+from os import path
+from typing import Dict
+
+import numpy as np
+
+# from api.models.student import Student
+from api.dataclasses.student import Student
+from benchmarking.data.interfaces import StudentProvider
+from manual_run.attributes import Attributes
+from manual_run.map_columns import MapColumns
+from manual_run.variables import Variables
+
+
+class DataProvider(StudentProvider):
+    def __init__(self):
+        self._sid_map: Dict[int, str] = {}
+        self._num_students = Variables.num_students
```

Comment:
```suggestion
```

nitpick;

No need to set self._num_students if it's only used in 1 place. See the code suggestion below